### PR TITLE
Assignment 1 completed

### PR DESCRIPTION
**Pull request by RossMalcolm**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Implementing a neural network to analyze the CFIAR-100 dataset

## What did you learn from the changes you have made?
I really developed a good understanding of CNN base and advanced features.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
I thought of many different approaches including Batch normalization, using different optimizers and changing dropout rates and regularization levels. However going down each of these rabbit holes seemed to make my model worse.

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
Many challenges of breaking my model after different parameter tweaking, but ultimately tried over and over again until I found what worked best for me.

## How were these changes tested?
Tested on google colab

## Checklist
- [X] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by vishnouvina — APPROVED** [review]
Approved A1
